src/context.h

```diff
@@ -6,7 +6,6 @@
 #define BITCOIN_CONTEXT_H
 
 #include <variant>
```

Comment:
d34ab741269e0042584584d022c4dce7a7f7a596: you need `<functional>` for `reference_wrapper`
```suggestion
#include <variant>
#include <functional>
```